lib/Conversion/TTIRToTTIRDecomposition/TTIRToTTIRDecomposition.cpp

```diff
@@ -233,14 +234,20 @@ struct ConvolutionToConv2dPattern
   constexpr static uint32_t SPATIAL_DIM_WIDTH = 1;
 
   // NHWC
-  const std::vector<int64_t> conv2dLayout = {
-      ConvolutionDimension::BATCH, SPATIAL_DIM_HEIGHT, SPATIAL_DIM_WIDTH,
-      ConvolutionDimension::FEATURE};
+  static inline const std::vector<int64_t> conv2dLayout = {
+      ConvolutionDimension::BATCH,
+      SPATIAL_DIM_HEIGHT,
+      SPATIAL_DIM_WIDTH,
+      ConvolutionDimension::FEATURE,
+  };
   // OIHW
-  const std::vector<int64_t> conv2dKernelLayout = {
+  static inline const std::vector<int64_t> conv2dKernelLayout = {
```

Comment:
why is `inline` required?

Reply:
- clang-tidy is mad about `const` data members, which are generally not a good pattern, they should be made static in 99% of cases.
- By making it `static` you then must define the member outside the class definition: https://en.cppreference.com/w/cpp/language/static
- I was too lazy and slightly preferred having the definition inline so I marked it `inline`